app/views/content/train-to-be-a-teacher/if-you-have-a-degree.md

```diff
@@ -28,102 +31,58 @@ keywords:
 ---
 
 
-You need a bachelor's degree in any subject to teach in primary, secondary and special schools in England.
+You need a bachelor's degree (in any subject) to teach in primary, secondary and special schools in England.
 
 For secondary teacher training, your provider will want to make sure you have a good knowledge of the subject you’re applying to teach.
 
-You also need to gain qualified teacher status (QTS) to teach in most schools which you get through teacher training.
+If you have a degree, you can do postgraduate teacher training to get qualified teacher status (QTS).
 
-Teacher training courses usually take 9 months full-time, or 18 to 24 months part-time.
+You need [English qualified teacher status (QTS)](/what-is-qts) to teach in most schools in England.
 
-If you’re not from the UK, find out about [training to teach in England as a non-UK citizen](/non-uk-teachers).
+$international-content$
 
-## What qualifications can I get through teacher training?
+## Teacher training course providers
 
-To be a qualified teacher, you can do teacher training to get:
+Your postgraduate teacher training course might be provided by: 
 
-* qualified teacher status (QTS)
-* qualified teacher status (QTS) with postgraduate certificate in education (PGCE)
+* a university (sometimes referred to as university-led training) 
+* a school or group of schools (sometimes referred to as school-led training, or an apprenticeship) 
+* [Teach First](https://www.teachfirst.org.uk/) (a charitable organisation)
 
-### QTS
+Some providers are ‘accredited’ – this means they've been given permission by the Department for Education (DfE) to run teacher training courses.
```

Comment:
Included this section (reworded with the help of Kate) because candidates will encounter this kind of information when they go into Find, so I thought it was important to address on GIT to help them understand what any of this means!!

Reply:
I think it is a good idea to include - is `permission` the right word though?

Might be worth checking with the accreditation team

Reply:
@Kateharries @MaeveRoseveare is 'permission' right here?

Reply:
maybe 'approved'?